cf-231A.mdx

```diff
@@ -5,7 +5,7 @@ title: Team
 author: Sathvik Chundru
```

Comment:
this file should be inside the solutions folder

Reply:
done

Reply:
i also moved it into expected knowledge